javascript/core/src/verbs/merge.ts

```diff
@@ -26,13 +26,33 @@ export interface MergeArgs extends InputColumnListArgs, OutputColumnArgs {
 	 * If it is not supplied, the values are just mashed together.
 	 */
 	delimiter?: string
+	unhot?: boolean
```

Comment:
Please add some additional docs for these params:
- unhot: Input columns have been one-hot encoded, the merge should extract the original value from the column name
- prefix: When using `unhot`, this is a shared column name prefix to remove from the value
- keepOriginalColumns: Merging eliminates the original source columns by default, collapsing into one. Set this to `true` to return the sources.